storage/accumulator/src/lib.rs

```diff
@@ -316,6 +332,38 @@ where
 
         Ok(AccumulatorProof::new(siblings))
     }
+
+    /// Implementation for public interface `MerkleAccumulator::get_consistency_proof`.
+    fn get_consistency_proof(&self, other_num_leaves: u64) -> Result<AccumulatorConsistencyProof> {
+        ensure!(
+            other_num_leaves <= self.num_leaves,
+            "The other accumulator is bigger than this one. self.num_leaves: {}. \
+             other_num_leaves: {}.",
+            self.num_leaves,
+            other_num_leaves,
+        );
+
+        // If the other accumulator is empty. Nothing is needed for the proof since any accumulator
+        // is consistent with an empty one.
+        if other_num_leaves == 0 {
+            return Ok(AccumulatorConsistencyProof::new(vec![], vec![]));
+        }
+
+        let frozen_subtree_roots = FrozenSubTreeIterator::new(other_num_leaves)
+            .map(|p| self.get_hash(p))
+            .collect::<Result<Vec<_>>>()?;
+
+        let root_level = Position::get_root_position(self.num_leaves - 1).get_level();
```

Comment:
It took a minute to realize the -1 is because we start indexing from 0.

It seems somewhat dangerous and arbitrary to be using bare integers here.

Reply:
lol, it took me 3 seconds.